imma254/so-lite
# API endpoints and tests for the StackOverflow-lite project

##What does this PR do?
Completes all the UI pages required for StackOverflow-Lite

##Description of tasks to be completed
Allow users to create accounts
Allow users to sign in to the created accounts
Allow users to view all questions(Click log in or sign up)
Allow users to answer a question
Allow users to ask a question
Allow users to view their profile

##How should this be manually tested?
https://github.com/imma254/so-lite/

Test	             				                API-endpoint					HTTP-Verbs
User can post a question			/api/v1/questions				POST
User can view all questions			/api/v1/questions				GET
User can view a single question		/api/v1/questions/question_id	GET
User can edit a question			        /api/v1/questions/question_id	PUT

##How should this manually be tested?
Navigate to the so-lite repository.
Clone or download the repository

$ . venv/bin/activate
$ pip install -r requirements.txt
$ python run.py

##Any background context you want to provide? **
Data structures

##What are the relevant Pivotal tracker stories? 
159842578
